### DISCUSSION TITLE
A simple example with nextjs 13

### DISCUSSION THREAD

**Discussion by nooblyf** [fabricjs/fabric.js#8540] [Q&A] [answered]
Can't seem to get `fabric.js` to work with nextjs 13, I've tried dynamic imports but no luck. I'm getting this error

```
Error: Module not found: Can't resolve '../build/Release/canvas.node'
```

I'm using "use client" so it's not a server component

<img width="956" alt="Screenshot 2022-12-28 at 19 10 10" src="https://user-images.githubusercontent.com/54323745/209820975-42c87152-f8a1-48ca-b153-dede03cf7219.png">

**Comment by ShaMan123** [comment, marked as answer]
This is a known and discussed issue
#8444 #8208 #8258

**Reply by ShaMan123** [reply]
Try this out:

[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/fabricjs/fabric.js/pull/8135)
